doc/mgr/rgw.rst

```diff
@@ -139,3 +139,32 @@ Join an existing realm by creating a new secondary zone (using the realm token)
   ceph rgw admin [*]
 
 RGW admin command
+
+Automatic usage log trimming
+----------------------------
+
+Enable automatic usage log trimming for one or more RGW realms by
+setting the config option ``usage_trim_realms`` to a comma separated
```

Comment:
what about `realms_to_trim`? because `trim_realms` seems like an action!

Reply:
That makes sense, I will change that.